lms/js_config_types.py

```diff
@@ -4,7 +4,28 @@
 Making this a top level module to avoid circular dependency problems.
 """
 
-from typing import NotRequired, TypedDict
+from typing import Literal, NotRequired, TypedDict
+
+
+class AutoGradingConfig(TypedDict):
+    grading_type: Literal["all_or_nothing", "scaled"] | None
+    """
+    - all_or_nothing: students need to meet a minimum value, making them get
+                      either 0% or 100%
+    - scaled: students may get a proportional grade based on the amount of
+              annotations. If requirement is 4, and they created 3, they'll
+              get a 75%
+    """
+
+    activity_calculation: Literal["cumulative", "separately"] | None
```

Comment:
Combined? We could always change it later.

Reply:
We could change it later, but ideally, before we ship anything to end users and start storing values that become outdated.

I would give the product team a couple of if days to make a decision

Reply:
BTW, the suggestion from a person who actually speaks English is to use `separate` rather than `separately` https://github.com/hypothesis/lms/pull/6632#discussion_r1743333154

Reply:
> separate

I might be able to actually spell that correctly so it works for me!

Reply:
Changed it to separate